compensated-ruby/bin/setup-matrix

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
Nit: This is also missing a top-level comment describing the purpose of the script.